modules/apps/frontend-taglib/frontend-taglib/src/main/resources/META-INF/resources/feature_indicator/index.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * SPDX-FileCopyrightText: (c) 2000 Liferay, Inc. https://liferay.com
+ * SPDX-License-Identifier: LGPL-2.1-or-later OR LicenseRef-Liferay-DXP-EULA-2.0.0-2023-06
+ */
+
+import {
```

Comment:
We shouldn't need a new React component for this. Instead in the JSP you should be able to just call our feature indicator component like this:
```
<react:component
	module="{FeatureIndicator} from frontend-js-components-web"
	props='<%=
		HashMapBuilder.<String, Object>put(
			"interactive", "true"
		).put(
			"learnResourceContext", LearnMessageUtil.getReactDataJSONObject("frontend-js-components-web")
		).put(
			"type", "deprecated"
		).build()
	%>'
/>
```

Reply:
Thanks for the tips @ethib137 ! I'm having trouble importing `com.liferay.learn.LearnMessageUtil`. It's giving the error `error: package com.liferay.learn does not exist`. You know what I'm missing by chance?

Reply:
You should just need to add the dependency to your build.gradle like [this](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-js/frontend-js-components-sample-web/build.gradle#L9). 

I also don't think you'll need `<% @taglib uri="http://liferay.com/tld/learn" prefix="liferay-learn" %>`. I think you only need the Util.

Reply:
Thanks @ethib137 for the missing link. I'm adding a couple more attributes on top of the ones you mentioned. I think we need to add `learnMessageResourceKey` and `learn ResourceContextJSONObject` so modules with custom messages can leverage this tag. I hit some typescript errors that I need to look at tomorrow.

Reply:
Hey @pat270 , we shouldn't need to add attributes for learnMessageResourceKey and learn ResourceContextJSONObject because this shouldn't allow custom messages. Each feature indicator type should only have one message that is always seen, no matter where the feature indicator is placed. Does that make sense?

Reply:
@ethib137 I see. I suppose we can add those later if we need it.